### ISSUE TITLE
encfs reverse - Filename too small to decode and Invalid data size, not multiple of block size

### ISSUE THREAD

**Issue by dakkusingh** [vgough/encfs#121]
Hi All,
I am running encfs on OSX 10.11.1 using brew.. encfs version 1.8.1
I created a Reverse ENCFS but i get a lot of errors like:

```
(encfs.cpp:138) withFileNode: error caught in getattr: Filename too small to decode
```

and 

```
(encfs.cpp:138) withFileNode: error caught in getattr: Invalid data size, not multiple of block size
```

Could someone help please?

**Comment by rfjakob** [comment]
Does it work otherwise? This is perfectly normal.

**Comment by dakkusingh** [comment]
it appears to work fine other than the errors. But as the errors worried me somewhat, I have not tested it fully

**Comment by vgough** [comment]
Yes, they are normal.  These errors should be dropped, or at a minimum filter out the expected patterns.

**Comment by hexadecagram** [comment]
If these messages are in fact benign (when using --reverse), why are they sent to syslog with LOG_ERR priority and not LOG_WARNING or better yet LOG_INFO?

**Comment by benrubson** [comment]
You're right they could be sent with LOG_WARNING priority.
However would need quite a few changes in the code to make this properly, as these messages come from raised exceptions.
Which files produces these messages ? .DS_Store on Mac OS ? Other files ?
Configuration file .encfs6.xml also produces this message in reverse mode, but #478 will solve this.

**Comment by hexadecagram** [comment]
1. I am seeing this behavior on ArchLinux, not OSX.
2. To my knowledge, there aren't any **particular** files that produce these messages. Syslog is being flooded with them as Crashplan scans the entire reversed encfs path recursively and backs up all of the files to the cloud. If I had to guess, I would be seeing the same pair of log messages for any and all files in that path.
3. The underlying filesystem that I am mounting encfs --reverse on top of is ZFS.
